goentri.com.clickmax.json

```diff
@@ -17,6 +17,11 @@
       "host": "@",
       "pointsTo": "%cnamePointsTo%",
       "ttl": 3600
+    },
+    {
+      "type": "REDIR301",
+      "target": "%target%",
+      "host": "@"
```

Comment:
no a chance this template would work. CNAME cannot co-exist with A/AAAA records needed for the redirect on the same host.

Reply:
Would this work if the host required is set to false and the CNAME host is back to www?

Reply:
yes, if you place CNAME on "www" then host required can be set to false

Reply:
it's done.